app/notifications/recipients/repository_row_creator.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class Recipients::ItemCreatorRecipients
+class Recipients::RepositoryRowCreator
```

Comment:
I think the LowStockReminder class also uses this, check if it also needs to be renamed there.

Reply:
I already set it in notification_extends:

 ```
   item_low_stock_reminder: {
      recipients_module: :RepositoryRowCreator
    },
```